src/libraries/System.Drawing.Common/src/System/Drawing/Printing/DefaultPrintController.cs

```diff
@@ -57,9 +58,10 @@ public override void OnStartPrint(PrintDocument document, PrintEventArgs e)
         /// <summary>
         /// Implements StartPage for printing to a physical printer.
         /// </summary>
-        public override Graphics OnStartPage(PrintDocument document, PrintPageEventArgs e)
+        public override Graphics? OnStartPage(PrintDocument document, PrintPageEventArgs e)
```

Comment:
Doesn't seem to return nullable

Reply:
Good point, although it doesn't change much since the base method is nullable.